[2024-07-30T11:09:41.138+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T11:09:41.184+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: enrich_prod.cleaning.check_esco_annexe_sensor manual__2024-07-30T11:09:39.166602+00:00 [queued]>
[2024-07-30T11:09:41.197+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: enrich_prod.cleaning.check_esco_annexe_sensor manual__2024-07-30T11:09:39.166602+00:00 [queued]>
[2024-07-30T11:09:41.198+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T11:09:41.219+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonSensor): cleaning.check_esco_annexe_sensor> on 2024-07-30 11:09:39.166602+00:00
[2024-07-30T11:09:41.227+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=2731) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-30T11:09:41.229+0000] {standard_task_runner.py:64} INFO - Started process 2744 to run task
[2024-07-30T11:09:41.231+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'enrich_prod', 'cleaning.check_esco_annexe_sensor', 'manual__2024-07-30T11:09:39.166602+00:00', '--job-id', '369', '--raw', '--subdir', 'DAGS_FOLDER/gps/custom_dags/enrich.py', '--cfg-path', '/tmp/tmps74776yx']
[2024-07-30T11:09:41.238+0000] {standard_task_runner.py:91} INFO - Job 369: Subtask cleaning.check_esco_annexe_sensor
[2024-07-30T11:09:41.304+0000] {task_command.py:426} INFO - Running <TaskInstance: enrich_prod.cleaning.check_esco_annexe_sensor manual__2024-07-30T11:09:39.166602+00:00 [running]> on host 7d7c82e77e1e
[2024-07-30T11:09:41.435+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='richmond.kongo@orange.com,assane.diop@orange.com,cyrille1.danho@orange.com,mohamed2.sylla@orange.com,amadou.diomande@orange.com,Jonas.kouame@orange.com,armand.kablan@orange.com,anicet.azombade@orange.com,etienne.kouadio@orange.com,bertine.kouakou@orange.com,florentine.kouakou@orange.com,ange.phaceli@orange.com,soulegame.soro@orange.com,innocent.yengo@orange.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='enrich_prod' AIRFLOW_CTX_TASK_ID='cleaning.check_esco_annexe_sensor' AIRFLOW_CTX_EXECUTION_DATE='2024-07-30T11:09:39.166602+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-30T11:09:39.166602+00:00'
[2024-07-30T11:09:41.436+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T11:09:41.456+0000] {python.py:75} INFO - Poking callable: <function check_file at 0x798a997f7a60>
[2024-07-30T11:10:11.657+0000] {local_task_job_runner.py:313} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-07-30T11:10:11.659+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-30T11:10:11.663+0000] {process_utils.py:132} INFO - Sending 15 to group 2744. PIDs of all processes in the group: [2744]
[2024-07-30T11:10:11.664+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 2744
[2024-07-30T11:10:11.665+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-30T11:10:11.666+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T11:10:11.717+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2744, status='terminated', exitcode=0, started='11:09:40') (2744) terminated with exit code 0
